channel/src/main/java/com/synopsys/integration/alert/channel/jira/cloud/util/JiraCloudIssueHandler.java

```diff
@@ -60,7 +61,8 @@ public JiraCloudIssueHandler(IssueService issueService, JiraCloudProperties jira
 
     @Override
     public IssueResponseModel createIssue(String issueCreator, String issueType, String projectName, IssueRequestModelFieldsMapBuilder fieldsBuilder) throws IntegrationException {
-        return issueService.createIssue(new IssueCreationRequestModel(issueCreator, issueType, projectName, fieldsBuilder, Collections.emptyList()));
+        IssueCreationResponseModel issueCreationResponseModel = issueService.createIssue(new IssueCreationRequestModel(issueCreator, issueType, projectName, fieldsBuilder, Collections.emptyList()));
+        return issueService.getIssue(issueCreationResponseModel.getKey());
```

Comment:
How did we get an `IssueResponseModel` from the service before this? Was it calling `getIssue` after creation internally?

Reply:
IssueCretionResponseModel is newly added. We were originally returning IssueResponseModel but pretty much all the values were empty. except 3-4 of them. So I created this new model to reflect that actual content.